iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeProcessor.java

```diff
@@ -429,10 +428,13 @@ public void fileNodeRecovery() throws FileNodeProcessorException {
           .put(FileNodeConstants.FILENODE_PROCESSOR_FLUSH_ACTION, flushFileNodeProcessorAction);
       String baseDir = directories
           .getTsFileFolder(newFileNodes.get(newFileNodes.size() - 1).getBaseDirIndex());
-      LOGGER.info(
-          "The filenode processor {} will recovery the bufferwrite processor, "
-              + "the bufferwrite file is {}",
-          getProcessorName(), fileNames[fileNames.length - 1]);
+      if (LOGGER.isInfoEnabled()) {
```

Comment:
The usage of LOGGER.ingo here is different from others. It's better to unify them.

Reply:
This is because the log here involves value evaluation. Adding an if can avoid unnecessary  value evaluation, which is not needed everywhere.

Reply:
> The usage of LOGGER.ingo here is different from others. It's better to unify them.

Remember that when you write `logger.info("xxx {} xxx", fun())`, then fun() will be executed no matter what log level you claim in the logback.xml. 

So, if fun() is a time-consuming function, you MUST wrap logger.SOME_LEVEL() with if(logger.isSOME_LEVELEnabled()), e.g., if(logger.isInfoEnabled())